jina/types/sets/chunk.py

```diff
@@ -30,7 +30,8 @@ def append(self, document: 'Document', **kwargs) -> 'Document':
 
         if not chunk.mime_type:
             chunk.mime_type = self._ref_doc.mime_type
-        chunk.update_id()
+        #chunk.update_id()
```

Comment:
What's the reason behind leaving these comments around?

Reply:
reminder to make sure, that one is completely refactored. Will be removed before merging. Good that you point it out.